BigQueryProvider/BigQueryConnection.cs

```diff
@@ -29,16 +29,7 @@ public class BigQueryConnection : DbConnection, ICloneable {
 
         public override void ChangeDatabase(string databaseName) {
             CheckDisposed();
```

Comment:
from the msdn:
IDbConnection.ChangeDatabase - Changes the current database for an open Connection object.

now it changes database regardless of ConnectionState